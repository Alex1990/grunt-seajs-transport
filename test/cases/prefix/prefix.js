define(function(require, exports, module) {
    var $ = require("jquery");

    $.fn.slider = function(elem, opts) {
        alert("init");
    };

});