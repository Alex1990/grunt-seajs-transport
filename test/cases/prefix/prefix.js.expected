define("jcn-prefix", ["jquery"], function(require, exports, module) {
    var $ = require("jquery");

    $.fn.slider = function(elem, opts) {
        alert("init");
    };

});